relayer-cli/src/commands/query/packet.rs

```diff
@@ -224,9 +255,29 @@ impl Runnable for QueryUnreceivedPacketsCmd {
         let dst_chain = CosmosSDKChain::bootstrap(dst_chain_config, rt).unwrap();
 
         // get the channel information from source chain
-        let channel = src_chain
+        let channel_res = src_chain
             .query_channel(&opts.port_id, &opts.channel_id, Height::zero())
-            .unwrap();
+            .map_err(|e| Kind::Query.context(e));
+
+        let channel = match channel_res {
+            Ok(c) => c,
+            Err(e) => {
+                return Output::error(format!(
+                    "failed to find the target channel ({}/{}) on src chain ({}) with error: {}",
+                    opts.port_id,
+                    opts.channel_id,
+                    src_chain.config().id,
+                    e
+                ))
+                .exit();
+            }
+        };
+
+        info!(
+            "Fetched from source chain {} the following channel {:#?}",
```

Comment:
This distracts a bit (make it debug?) and the pretty format doesn't show well. 
```suggestion
        debug!(
            "Fetched from source chain {} the following channel {:?}",
```